src/doc/src/reference/environment-variables.md

```diff
@@ -334,11 +334,15 @@ let out_dir = env::var("OUT_DIR").unwrap();
 * `RUSTC`, `RUSTDOC` — the compiler and documentation generator that Cargo has
                        resolved to use, passed to the build script so it might
                        use it as well.
+* `CARGO_RUSTC_WRAPPER` — the `rustc` wrapper, if any, that Cargo is using.
+                          See [`build.rustc-wrapper`].
 * `RUSTC_LINKER` — The path to the linker binary that Cargo has resolved to use
                    for the current target, if specified. The linker can be
                    changed by editing `.cargo/config.toml`; see the documentation
                    about [cargo configuration][cargo-config] for more
                    information.
+* `CARGO_RUSTFLAGS` — the `RUSTFLAGS` that Cargo invokes `rustc` with.
+                      See [`build.rustflags`].
```

Comment:
I expect this also incorporates `target.FOO.rustflags` and incoming env `RUSTFLAGS`, with all the host/target build separation that usually entails.

Reply:
I see that is spelled out in the docs for `build.rustflags` already.

Reply:
It should — the way I understand the current code flow is that this should include the final list of flags that Cargo _would_ pass to `rustc`, while taking all configuration into account.